src/ode/ode_sir_models.jl

```diff
@@ -0,0 +1,39 @@
+function SIR(du,u,p,t)
+    S = u[1] ; I = u[2] ; R = u[3];
+    β = p[1] ; γ = p[2] ; N = p[3]
+    du[1] = -β*S*I/N
+    du[2] = β*S*I/N - γ*I/N
+    du[3] = γ*I/N
+end
+p = [0.25,0.1,1000]
+u₀ = [999,1,0]
```

Comment:
I'm pretty sure you don't want integers here.

Apart from that, I guess it would also be cleaner to not redefine `p` etc., possibly by using `let` blocks.